test/augmentation/test_augmentation.py

```diff
@@ -4112,3 +4113,29 @@ def test_gradcheck(self, device):
         input = utils.tensor_to_gradcheck_var(input)  # to var
         # TODO: turned off with p=0
         assert gradcheck(kornia.augmentation.RandomAutoContrast(p=1.0), (input,), raise_exception=True, fast_mode=True)
+
+
+class TestRandomSnow:
```

Comment:
from kornia.testing 
```suggestion
class TestRandomSnow(BaseTester):
```

check https://github.com/kornia/kornia/blob/master/CONTRIBUTING.md#coding-standards